source/xml/writing.md

```diff
@@ -122,13 +122,14 @@ This could output:
 You can even mix these syntaxes:
 
     $ns = '{http://www.w3.org/2005/Atom}';
-    $writer->namespaceMap['http://www.w3.org/2005/Atom'] = '';
```

Comment:
this fixes the `:` problem described in https://github.com/fruux/sabre-xml/pull/61#issuecomment-162186171